src/main/java/gov/nasa/pds/validate/ri/OpensearchDocument.java

```diff
@@ -162,6 +164,8 @@ protected SearchResponse search(RestHighLevelClient client, SearchRequest reques
 
   public OpensearchDocument(AuthInformation context) {
     this.context = context;
+    this.hugeMemory.setHttpAsyncResponseConsumerFactory(
+        new HttpAsyncResponseConsumerFactory.HeapBufferedResponseConsumerFactory(128*1024^4)); // 128 GB
```

Comment:
The `^` in Java is the bitwise exclusive-OR operator, right? And it has lower precedence than `*`, so this becomes

    (128 * 1024) ^ 4 = 131072 ^ 4 = 0x20000 ^ 0x4 = 0x20004 = 131076

which isn't too close to 128 GB 😊

I think for 128 GB you want `128 * 1024 * 1024 * 1024` (three, not four—otherwise it'd be terabytes, not gigabytes).

Reply:
You are correct. Too many languages. I tried `**` and had syntax error so just mapped to `^` and forgot that Java does not know how to do exponential since it is a CS language.